packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -530,7 +530,6 @@ name: foo
     'Dart2JSTarget calls dart2js with expected args with csp',
     () => testbed.run(() async {
       environment.defines[kBuildMode] = 'profile';
-      environment.defines[JsCompilerConfig.kCspMode] = 'true';
```

Comment:
I searched for any reference to this const and below. No references anywhere in code (except tests). Never read.